ansible/roles/common_installs/tasks/main.yml

```diff
@@ -33,6 +33,11 @@
   sudo: yes
   with_items:
     - less@1.3.1
+    - n@1.3.0
```

Comment:
The comment stream on that page you linked to is less than inspiring. Isn't there a way to do this with the system package manager (and doesn't ansible know all about such things)?